packages/core/src/delegation/DelegationNode.ts

```diff
@@ -398,7 +407,8 @@ export class DelegationNode implements IDelegationNode {
   public async getRemoveTx(): Promise<SubmittableExtrinsic> {
     const childCount = await this.subtreeNodeCount()
```

Comment:
this would actually be a good place to help out with another custom rpc

Reply:
A ticket would be great, but let's talk priority in the next planning

Reply:
Looks like not an issue with the PR itself :)